organization/_posts/2016-01-21-startup.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Startup Team Minutes #42"
+layout: default
+---
+
+# HEP Software Foundation Startup Team Meeting notes Jan 21 2016
+
+[Indico Agenda](https://indico.cern.ch/event/487072/)
+
+Present: Andrea, Andrew, Benedikt, Dario, Liz, Michel
+
+Apologies from: Pere, Torre
+
+## News and general updates
+- 2nd conditions data meeting between ATLAS/CMS/Belle-II took place yesterday
+ ( [Indico Agenda](https://indico.cern.ch/event/483692/) ). All parties seem in
+ evaluation stage of new technologies/approaches. Strong attendance and motivation
+ to follow up on the newly created mailing list. Andrea makes a remark about
+ that we need to advertise it. More information to add to the website.
+
+## Technical Notes
+- New technical note on results of WLCG MJF Task Force almost finished. Once
+final authorlist is defined, it will become official.
+- Benedikt suggests to ask Tim Smith whether he wants to give feedback to the
+license TN. If not, the note is considered final.
+
+## Training
+- Discussing again the problem of finding volunteers to contribute material.
+- Dario contacted people from the SFT group. Needs a follow up with Pere.
+- Benedikt contacted a few people from CMS and the German community.
+- Announcements of training activities are not distributed properly.
+Should the HSF
+try to be more active in distributing information?
+- Dario would like to step-down once a successor is found.
```

Comment:
Was it that Daria "would like" or "is ready"? I think he said he would like to see new people involved and his ready to step down if needed...